HIP/hip-0000-event-serialization.md

```diff
@@ -0,0 +1,227 @@
+---
+hip: <HIP number (this is determined by the HIP editor)>
+title: Event Serialization With Round Number, EventDescriptors, and Multiple Other Parents.
+author: Edward Wertz <edward@swirldslabs.com>
+working-group: Swirlds Labs Platform Hashgraph Team
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+created: 2023-10-25
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/827
+updated: <comma separated list of dates>
+requires: <HIP number(s)>
+replaces: <HIP number(s)>
+superseded-by: <HIP number(s)>
+---
+
+## Abstract
+
+This HIP proposes several changes to the event serialization format.
+
+1. Add the round during which an event was created to the event's meta-data.
+2. Represent links to event parents as EventDescriptors, 4-tuples of (Event Hash, Creator NodeId, Generation, Round
```

Comment:
Should this say `Roster Round` instead of `Round Number`? (also mentioned elsewhere in this doc)

Reply:
I've modified the text to eliminate reference to address book round.  Please re-review the text.